dds/idl/marshal_generator.cpp

```diff
@@ -451,6 +451,24 @@ namespace {
     }
 
     AST_Type* elem = resolveActualType(seq->base_type());
+    AST_Annotation_Appl* ann_appl = seq->base_type_annotations().find("::@try_construct");
```

Comment:
I'm pleasantly surprised you figured out how to do this. It might be a good idea to add a helper method for this to the class from `annotations.h`.

Reply:
Sorta like how there's a `KeyAnnotation::union_value`, for example.

Reply:
To be more clear, the way annotations are currently handled is that they are defined in classes in `annotation.h/cpp` and then accessed using methods on `be_global` to allow for the possibility of some setting changing the behavior.